app/models/miq_group.rb

```diff
@@ -124,7 +128,7 @@ def self.get_httpd_groups_by_user(user)
     rescue => err
       raise _("Unable to get groups for user %{user_name} - %{error}") % {:user_name => username, :error => err}
     end
-    user_groups.first
+    strip_group_domains(user_groups.first)
```

Comment:
@chessbyte Yes user_groups.first returns an array with a single element which itself is an array. I have  tested this change on a live appliance.